lib/workers/repository/update/pr/changelog/source-github.ts

```diff
@@ -53,7 +54,7 @@ export async function getChangeLogJSON({
   });
   // istanbul ignore if
   if (!config.token) {
-    if (host.endsWith('github.com')) {
+    if (host.endsWith('github.com') && GlobalConfig.get().githubTokenWarn) {
```

Comment:
I think the section below this could benefit from some extra if/else logic so that the debug message which is printed in your case makes sense. It could be the same message as the warn, and then return null.

Reply:
@rarkins good catch, updated!